ddl/ddl_worker.go

```diff
@@ -393,6 +393,11 @@ func (d *ddl) runDDLJob(t *meta.Meta, job *model.Job) (ver int64, err error) {
 
 		job.Error = toTError(err)
 		job.ErrorCount++
+
+		metrics.DDLJobErrCounter.WithLabelValues(metrics.LabelDDL).Inc()
```

Comment:
There is already a metric [here](https://github.com/pingcap/tidb/blob/master/ddl/ddl.go#L458).

Reply:
There are different metrics. Here collects the count of errors in ddl_worker.go that may not return the error in https://github.com/pingcap/tidb/blob/master/ddl/ddl.go#L458.